lldb/source/Target/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -292,7 +292,7 @@ class LLDBMemoryReader : public swift::remote::MemoryReader {
       } else {
         *result = 0;
       }
-      break;
+      return true;
```

Comment:
Swift reflection code expects this function to return true when a valid value is assigned to `outBuffer`.